pkg/gadgets/run/types/types.go

```diff
@@ -19,15 +19,34 @@ import (
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
+type Endpoint struct {
+	eventtypes.L4Endpoint
+	Version uint8  `json:"version,omitempty"`
```

Comment:
Other PR: Move the `Version` field into the `L3Endpoint`?

Reply:
I did in this one as part of the reworking to have types for L3 and L4 endpoints in C.